### ISSUE TITLE
Add networking code

### ISSUE THREAD

**Issue by hvge** [wultra/swift-lime-auth#21]
We should add a simple networking code to the library with following properties:

- Networking part should be optional, implemented as a "subspec" addon
- With support for PowerAuth signatures
- With support for PowerAuth token signatures
- With support for PowerAuth ECIES encryptor (may be added later, but we should count with this in the library design)

**Comment by hvge** [comment]
Partially blocked by lime-company/powerauth-mobile-sdk#121

**Comment by hvge** [comment]
Closing. I think that this feature is no longer relevant for us.
